### ISSUE TITLE
[BUG] Includes parameter doesn't seem to be working with Get-PnPFolder

### ISSUE THREAD

**Issue by PowershellScripts** [pnp/powershell#2595]
### Notice
I don't have the skill to assess if the issue lies in https://github.com/pnp/powershell/blob/c13d935686dcac6c3949e8fb7861b5ce26135110/src/Commands/Base/PnPRetrievalsCmdlet.cs or somewhere else

### Reporting an Issue or Missing Feature
When trying to load additional properties for a folder using `Get-PnPFolder` and `-Includes` parameter, they don't seem to be loaded. It works if the same property is retrieved with `$context.load()`. Here the examples use `ParentFolder` property. I got the same behaviour with `ContentTypeOrder`,` Folders.Count`, or `Files`.

```
PS C:\WINDOWS\system32> $fWithInclude = Get-PnPFolder -List dokus -Includes ParentFolder
PS C:\WINDOWS\system32> $fWithInclude[0].ParentFolder
Name Type   Items/Size Last Modified
---- ----   ---------- -------------
     Folder    
```                     

### Expected behavior 
I'd expect the same behaviour as with `Get-PnPList -Includes`:

![image](https://github.com/PowershellScripts/shared/blob/main/get-pnpfolder2.png?raw=true)

### Actual behavior 
![image](https://github.com/PowershellScripts/shared/blob/main/get-pnpfolder.png?raw=true)

### Steps to reproduce behavior
```
$fWithInclude = Get-PnPFolder -List dokus -Includes ParentFolder
$fWithInclude[0].ParentFolder
```

### What is the version of the Cmdlet module you are running?
1.12.19

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by KoenZomers** [comment]
Thanks for reporting @PowershellScripts. It has been fixed through PR #2735 .

**Comment by gautamdsheth** [comment]
Thanks @PowershellScripts for reporting this, it has been fixed and will be available in tomorrow's nightly build and the next major version whenever we release that :)
